bench/BENCHMARK_DEV.md

```
@@ -1,530 +1,531 @@
+  Reqs/sec    187696.99   20928.84  203109.58
+  Latency      518.00us   454.49us     6.18ms
   Latency Distribution
-     50%   497.00us
-     75%   602.00us
-     90%   712.00us
-     99%     1.61ms
+     50%   344.00us
+     75%   639.00us
+     90%     1.01ms
+     99%     2.76ms
 ### Redirect Response (/redirect)
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)